website/content/en/docs/deployment/cognito/manifest/guide.md

```diff
@@ -24,98 +26,133 @@ Read the [create domain and certificate section]({{< ref "/docs/add-ons/load-bal
-    1. In the `Domain name` choose `Use your domain`, type `auth.platform.example.com` and select the `*.platform.example.com` AWS managed certificate you’ve created in N.Virginia. Creating domain takes up to 15 mins.
-        1. ![cognito-active-domain](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/cognito-active-domain.png)
-        2. When it’s created, it will return the `Alias target` CloudFront address.
-            1. Screenshot of the CloudFront URL for Cognito Domain name:
-                1. ![cognito-domain-cloudfront-url](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/cognito-domain-cloudfront-url.png)
-            2.  Create a new record of type `A` for `auth.platform.example.com` with the value of the CloudFront URL.
-                1. Select the `alias` toggle and select Alias to Cloudfront distribution for creating the record
-                2. ![cognito-domain-cloudfront-A-record-creating](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/cognito-domain-cloudfront-A-record-creatin.png)
-                3. Following is a screenshot of the A record for `auth.platform.example.com` in `platform.example.com` hosted zone:
-                    1. ![cognito-domain-cloudfront-A-record-created](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/cognito-domain-cloudfront-A-record-created.png)
+
+    * In order to use a custom domain, its root(i.e. `platform.example.com`) must have an valid A type record. Create a new record of type `A` in the `platform.example.com` hosted zone with an arbitrary IP for now. Once the ALB created, we will update this value.
+    
+        Following is a screenshot of a `platform.example.com` hosted zone showing a record. 
+
+        ![subdomain-initial-A-record](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/subdomain-initial-A-record.png)
+
+    * If your cluster is not in N.Virginia(us-east-1), create an ACM certificate in us-east-1 for `*.platform.example.com` by following the steps in [section 2.0]({{< ref "/docs/add-ons/load-balancer/guide.md#create-certificates-for-domain" >}}). That is because [Cognito requires](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-add-custom-domain.html) a certificate in N.Virginia in order to have a custom domain for a user pool.
```

Comment:
looks like the section 2.0 has been renamed to just create certificate for domain. Please reword this accordingly